fuzz/cli.rs

```diff
@@ -223,6 +240,18 @@ fn run_afl(target: &str) -> Result<(), Error> {
 
 /// Run one target fuzz test using Honggfuzz
 fn run_honggfuzz(target: &str) -> Result<(), Error> {
+    let pre_check = Command::new("cargo")
+        .args(&["hfuzz", "version"])
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
+        .status()
+        .unwrap();
+    if !pre_check.success() {
+        Err(format_err!(
+            "cannot run hfuzz subcommand, please run `cargo install hfuzz --version 0.5.34` before testing"
+        ))?;
```

Comment:
Could you please extract the two version checking code as a standalone function (with cli args and the error messages as the parameters)?